op-node/rollup/derive/batch_stage.go

```diff
@@ -81,7 +81,7 @@ func (bs *BatchStage) NextBatch(ctx context.Context, parent eth.L2BlockRef) (*Si
 	switch validity {
 	case BatchAccept: // continue
 		batch.LogContext(bs.Log()).Debug("Found next singular batch")
-		return batch, len(bs.nextSpan) == 0, nil
+		return batch, true, nil // in Holocene, blocks immediately become safe
```

Comment:
This is the only logic change of this PR

Reply:
Wait, where is the `isHolocene` check? Doesn't this make all pre-holocene batches appear as safe too, while they should be pending-safe?

Reply:
It could be good to add this context as a comment

Reply:
improved the comment